apps/site/components/withMetaBar.tsx

```diff
@@ -20,6 +20,10 @@ const WithMetaBar: FC = () => {
   const lastUpdated = frontmatter.date
     ? formatter.dateTime(new Date(frontmatter.date), DEFAULT_DATE_FORMAT)
     : undefined;
+  const readingTimeText = formatter.number(Math.round(readingTime.minutes), {
```

Comment:
Why `Math.round`? As far as I know, `Intl.NumberFormat` rounds automatically.

Reply:
I’m using next-intl for this, is something wrong?

Reply:
Oh yeah I didn't see it. Oups my bad. So yeah we should try to avoid `Math`

Reply:
6cfa18a14b58211fb7aef2c59ee6876b4e4ed1c3

I'm not sure why we should avoid the standard Math functions, but I used the `maximumFractionDigits` option instead. How does this look?

Reply:
lgtm